src/Core/Style.js

```diff
@@ -895,6 +895,13 @@ class Style {
                 }
             }
         }
+        // VectorTileSet: by default minZoom = 0 and maxZoom = 24
+        // https://docs.mapbox.com/style-spec/reference/layers/#maxzoom and #minzoom
+        // Should be move to layer properties, when (if) VectorTile will be considered as multiLayer.
```

Comment:
A VectorTileSource send for each tile data on several 'layer' (building, road etc for instance), thus the use of the propertiy filter (that should be move to layer in my opinion). So one Layer with a VectorTileSource will have data on several vectorTile 'layer' that can have diferent 'layer' properties (style, zoom max and min etc). That why i fell like this kind of source is more a multiLayer that a real Layer.

Reply:
Ok I understand what you mean, there is itowns layer concept and mapbox layer concept; can you precise your comment then or open an issue please? Otherwise I'm afraid that we won't understand this comment in a few month...